delete/main.go

```diff
@@ -29,7 +29,8 @@ func main() {
 		a.Fatalf("failed to get GitHub context: %v", err)
 	}
 
-	do := godo.NewFromToken(in.token).Apps
+	do := godo.NewFromToken(in.token)
+	do.UserAgent = "do-app-action-delete"
```

Comment:
is there a way to add a version of the action here?

Reply:
Getting the proper version might be tricky, outside of manually updating it. I'll try to see if we can get the commit SHA added easily though.

Reply:
Tried for a bit and it seems like the action isn't checked out as a git repository. There's an `action_ref` field on the Github context that contains the reference the user used to fetch the action but that also feel kinda meh tbh.

The most reliable way would likely be to hardcode and keep updating it, though I also don't love that 😓 . I'll move forward as-is though, we can enhance this further down the line as we see fit.